lib/src/features/streammanagement/StreamManagementModule.dart

```diff
@@ -223,6 +222,16 @@ class StreamManagementModule extends Negotiator {
 
   bool isResumeAvailable() => streamState.isResumeAvailable();
 
+  void resetAckTimer() {
+    streamState = StreamState();
```

Comment:
@slashdigital here so i reset all of them when resume, because when resume, we need to open a new socket, so we cannot continue the number.